src/components/BidModal/BidModal.jsx

```diff
@@ -0,0 +1,124 @@
+import React, { useEffect, useRef, useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { S } from './BidModal.js';
+import OrderBidModal from '../BidOrderModal/BidOrderModal.js';
+
+const BidModal = ({
+  modalOpen,
+  setModalOpen,
+  detail,
+  setOrderOpen,
+  orderOpen,
+}) => {
+  const [coast, setCoast] = useState(0);
+  const [ticket, setTicket] = useState(1);
+  const inputRef = useRef();
+  const outside = useRef();
+
+  const minus = () => {
+    if (ticket > 1) {
+      return setTicket(ticket - 1);
+    } else {
+      return 1;
+    }
+  };
+
+  const plus = () => {
+    if (ticket == 4) {
+      return 4;
+    } else {
+      return setTicket(ticket + 1);
+    }
+  };
+
+  useEffect(() => {
+    document.body.style.cssText = `
+      position: fixed; 
+      top: -${window.scrollY}px;
+      overflow-y: scroll;
+      width: 100%;`;
+    return () => {
+      const scrollY = document.body.style.top;
+      document.body.style.cssText = '';
+      window.scrollTo(0, parseInt(scrollY || '0', 10) * -1);
+    };
+  }, []);
+
+  return (
+    <S.BidModalContainer ref={outside}>
+      <S.BidModalWrapper>
+        <S.CloseIcon onClick={() => setModalOpen(false)} />
+        <S.BidModalWrap>
+          <S.BidEventImg src={detail.thumbnail_images_url} alt="event" />
+          <S.BidContentWrap>
+            <S.BidTitle>{detail.name}</S.BidTitle>
+
+            <S.BidContent>{detail.event_start_date}</S.BidContent>
+            <S.BidContent> {detail.location}</S.BidContent>
+          </S.BidContentWrap>
+        </S.BidModalWrap>
+        <S.BidCountContainer>
+          <S.BidCountWrap>
+            <S.MinusIcon onClick={() => minus()} />
+            {ticket}
+            <S.PlusIcon onClick={() => plus()} />
```

Comment:
```suggestion
            <S.MinusIcon onClick={minus} />
            {ticket}
            <S.PlusIcon onClick={plus} />
```
인자로 전달해 주는 값이 없으면 위와같이 작성하셔도 좋습니다.